### ISSUE TITLE
UI5-Button | iOS | a11y -  Button Focus outline not visible

### ISSUE THREAD

**Issue by yoganjan** [SAP/ui5-webcomponents#8178]
### Bug Description

UI5-Button focus is not visible while tabbing. This is happening for iphone 

### Affected Component

Button

### Expected Behaviour

Outline focus should be visible for the button when its fouced while tabbing

### Steps to Reproduce

1. Open https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-button--docs
2. Open Toggle Device Tool Bar from chrome dev tool
3. Select iPhone 14 Pro Max 
4. Tray focusing the button by clicking on keyboard tab
...

### Priority

Medium

### UI5 Web Components Version

1.20.0

### Browser

Chrome

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by LidiyaGeorgieva** [comment]
Hello @yoganjan ,

If you are testing on real device (phone) there will be no "tab" key. The way you test seems wrong to me.
Also on phone there is no need of features as focus, hover, etc. since they are not useful without real keyboard or mouse.

Best Regards,
Lidiya

**Comment by yoganjan** [comment]
Hi @LidiyaGeorgieva  ,
We are testing by connecting blueetooth keyboard to the iphone and testing.  Same we were able to reproduce from chrome dev tools as well

**Comment by LidiyaGeorgieva** [comment]
Hello @yoganjan ,
Could you please explain which Accessibility standard is violated.
Best Regards,
Lidiya

**Comment by yoganjan** [comment]
Hi @LidiyaGeorgieva  ,

When we navigate the input fields with Tab key on keyboard , we should see outline for the buttons. But for the Ui5-Button outline is not visible when its focused using Tab 

Thanks,
Yoganjan

**Comment by LidiyaGeorgieva** [comment]
Hello @SAP/ui5-webcomponents-topic-b,
Please advice.
Regards,
Lidiya

**Comment by petyabegovska** [comment]
Note: This is part of bigger topic for visual focus on mobile: FIORITECHP1-17583.
